org.jasp.JASP.json

```diff
@@ -0,0 +1,430 @@
+{
+	"app-id": "org.jasp.JASP",
```

Comment:
do you also control the jasp.org DNS as well as jasp-stats.org? Ideally your app-id is under your control, but if this is what the app is coded to accept it could be OK.

Reply:
Well, we do not actually control jasp.org but I figured this was a bit cleaner to type and would be unlikely to conflict with anything. Is this a big problem?